components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/dao/SQLQueries.java

```diff
@@ -396,6 +423,10 @@ public class SQLQueries {
     public static final String GET_ROLES_BY_TENANT_AND_ROLE_NAME_TAIL_DB2 = " UM_TENANT_ID=:UM_TENANT_ID;) AS " +
             "AUDIENCE WHERE rn BETWEEN :ONE_BASED_START_INDEX; AND :END_INDEX;";
 
+    public static final String GET_USERS_ROLES_BY_TENANT_AND_ROLE_NAME_TAIL_DB2 =
+            "UM_HYBRID_USER_ROLE.UM_TENANT_ID=:UM_TENANT_ID) AS USERS WHERE rn BETWEEN :ONE_BASED_START_INDEX " +
+            "AND :END_INDEX ORDER BY rn";
```

Comment:
Is this intentional using `ORDER BY rn`, Just checking since in other places we use `ORDER BY UM_USER_NAME`